src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -40,6 +40,14 @@
         </li>
       </ul>
     </div>
+
+    <button class="btn btn-sm btn-default btn-label tc_configureCluster"
+            type="button"
+            (click)="configureClusterAction()">
+      <i class="fa fa-fw fa-cog"
+         aria-hidden="true"></i>
+      <ng-container i18n>Configure Cluster-wide OSD Flags</ng-container>
```

Comment:
How about `Set` instead of `Configure` as the actions a more boolean based.

Reply:
Done.